tests/e2e_tests/test_bp_map.py

```diff
@@ -0,0 +1,255 @@
+import argparse
+import os
+import pytest
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib import colors
+from astropy.io import fits
+from corgidrp import data
+from corgidrp import caldb
+from corgidrp import detector
+from corgidrp import darks
+from corgidrp import walker
+
+thisfile_dir = os.path.dirname(__file__)  # this file's folder
+
+@pytest.mark.e2e
+def test_bp_map(tvacdata_path, e2eoutput_path, use_master_dark):
+    """
+    Generate and test the bad pixel map using TVAC data.
+
+    Args:
+        tvacdata_path (str): Path to the TVAC data directory.
+        output_path (str): Directory to save the output files.
+        use_master_dark (bool): Whether to use a master dark or a simple dark.
+
+    Returns:
+        None
+    """
+    cals_dir = os.path.join(tvacdata_path, "Cals")
+
+    # Make output directory if needed
+    bp_map_outputdir = os.path.join(e2eoutput_path, "bp_map_output")
+    if not os.path.exists(bp_map_outputdir):
+        os.mkdir(bp_map_outputdir)
+    dark_current_filename = "dark_current_20240322.fits"
```

Comment:
all these filenames should match the filenames on Box.

Reply:
Echoing an above comment - there are issues with the files on Box. Once Marie is able to fix them I will update the names